spec/index.html

```diff
@@ -1994,8 +1996,7 @@ <h5 id="the-properties-element">4.2.18 The <code>&lt;properties&gt;</code> eleme
         </dl>
 
         <p>
-          A <code>feature</code> element can have zero or one <code>properties</code> element, which contains zero or more unknown elements, whose content is text. Note: it would be better to a content model of property child elements whose
-          values could be HTML. TODO.
```

Comment:
Removed this note, as there is already a "TODO allow HTML content." in the content model description.